robotiq_s_model_visualization/cfg/s-model_articulated_macro.xacro

```diff
@@ -56,12 +57,17 @@ there are multiple hands then a prefix followed by an "_" is needed.
 			<origin xyz="-0.0455 0.0214 -0.036" rpy="0 3.1416 -1.57"/>
 			<limit lower="-0.25" upper="0.16" effort="100" velocity="100"/>
 		</joint>
-		<joint name="${prefix}palm_finger_middle_joint" type="fixed">
+
+                <!-- WARNING: LIMITS NOT CORRECT -->
+                <!-- Changed to revolute so gazebo doesn't flip tables -->
```

Comment:
I'm fine with anything, as long as it is placed somewhere *outside* the xacro/urdf itself.